server/controllers/api_controller.go

```diff
@@ -163,10 +162,7 @@ func (a *APIController) apiPlan(request *APIRequest, ctx *command.Context) (*com
 		res := a.ProjectPlanCommandRunner.Plan(cmd)
 		projectResults = append(projectResults, res)
 
-		err = a.PostWorkflowHooksCommandRunner.RunPostHooks(ctx, cc[i])
-		if err != nil {
-			ctx.Log.Err("Error running post-workflow hooks %s.", err)
-		}
+		a.PostWorkflowHooksCommandRunner.RunPostHooks(ctx, cc[i]) // nolint: errcheck
```

Comment:
Not sure why I follow why these logs are being removed?

Reply:
They have been moved into `events/post_workflow_hooks_command_runner.go` rather than being duplicated here.

Reply:
Ah I see. Always a bit apprehensive when an `err` is being swallowed though; if we expect the body of RunPostHooks to "handle" the error (i.e. logging it), and we've now change all the call sites of this function to ignore the error, maybe it shouldn't return an error at all?

Reply:
The tests use the return error, so we should leave it there.

Reply:
Hi @lukemassa, any chance we can get this PR over the line? I'm not really bothered about de-duping these log messages, so if you feel strongly about this, I'll just revert my changes. I want to get the other enhanced logging through.

Reply:
I don't feel that strongly, no; leaving it in for the tests is good enough for me.